### DISCUSSION TITLE
Fashion-gram

### DISCUSSION THREAD

**Discussion by charvisethi** [Tech-Phantoms/Maintainers-Program#17] [Application]
About Me: I am Charvi, Sophomore IGDTUW. I am an upcoming android developer and a AR/VR enthusiast. I have worked on 2 projects one was the She-Assistant (https://github.com/charvisethi/She-Assistant.git) and an SparkAR filter Archangels ( https://www.instagram.com/ar/254847896628002). I am currently working with a startup ADto which involves development in Flutter.
LinkedIn: https://www.linkedin.com/in/charvi-sethi/
GitHub: https://github.com/charvisethi

**Comment by imabp** [comment]
👋 Hey  Charvi, Thank you for submitting the maintainers program form.
As discussed in the Project Readme, please add your open source project link here, that you want to put under Maintainers Program.

So we can review it and proceed accordingly.
